src/color.scss

```
@@ -0,0 +1,31 @@
+// Copyright (c) 2021 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+/* stylelint-disable custom-property-empty-line-before, declaration-no-important */
+
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+
+:host {
+  background-color: var(--ds-auro-menuoption-background-color);
+  color: var(--ds-auro-menuoption-text-color);
+}
+
+:host([disabled]) {
+  --ds-auro-menuoption-text-color: var(--ds-color-text-ui-disabled-default, $ds-color-text-ui-disabled-default);
+}
+
+:host(:hover:not([disabled])),
+:host(.active:not([disabled])) {
+  --ds-auro-menuoption-background-color: var(--ds-color-container-ui-secondary-hover-default, #{$ds-color-container-ui-secondary-hover-default});
+}
+
+:host([selected]:not([disabled])) {
+  --ds-auro-menuoption-background-color: var(--ds-color-container-ui-primary-default-default, $ds-color-container-ui-primary-default-default);
+  --ds-auro-menuoption-text-color: var(--ds-color-text-primary-inverse, $ds-color-text-primary-inverse);
+  
+  svg {
+    fill: var(--ds-auro-menuoption-icon-color) !important;
+  }
```

We don't render the SVG in the DOM if it's not selected so we can just set the token without looking for the attribute. Then we override the token when disabled.
```suggestion
:host {
  svg {
    fill: var(--ds-auro-menuoption-icon-color) !important;
  }
}
:host([disabled]) {
  svg {
    --ds-auro-menuoption-icon-color: transparent;
  }
}
```

---

This is not going to work because the `fill` rule is being set in a specific selector. See new solution.